packages-ts/starknet-gauntlet-ocr2/src/commands/ocr2/deploy.ts

```diff
@@ -20,13 +26,36 @@ type ContractInput = [
 
 const makeUserInput = async (flags, args, env): Promise<UserInput> => {
   if (flags.input) return flags.input as UserInput
+
+  if (flags.rdd) {
+    const rdd = getRDD(flags.rdd)
+    const contractAddr = args[0]
+    const aggregator = rdd[CONTRACT_TYPES.AGGREGATOR][contractAddr]
+    return {
+      maxAnswer: aggregator.maxSubmissionValue,
+      minAnswer: aggregator.minSubmissionValue,
+      decimals: aggregator.decimals,
+      description: aggregator.name,
+      billingAccessController:
+        aggregator.billingAccessController ||
+        flags.billingAccessController ||
+        env.billingAccessController ||
```

Comment:
`billingAccessController` was commonly referred as `BILLING_ACCESS_CONTROLLER` in the env; `linkToken` as `LINK`. Please check with the gauntlet team and/or their codebase for the default values, let's stay consistent.

Reply:
good catch, fixed thanks